src/main/java/trycb/web/TenantUserController.java

```diff
@@ -88,7 +88,7 @@ public TenantUserController(Bucket bucket, TenantUser tenantUserService, TokenSe
             jwtService.verifyAuthenticationHeader(authentication, username);
             Result<Map<String, Object>> result = tenantUserService.registerFlightForUser(bucket, tenant, username,
                     jsonData.getArray("flights"));
-            return ResponseEntity.accepted().body(result);
+            return ResponseEntity.ok().body(result);
```

Comment:
is this related to Swagger?

Reply:
Yeah, we return 200 in the swagger definition, so just updating to make sure we're doing the same here.